client/src/app/site/deployment-center/deployment-center-setup/step-configure/configure-vsts-source/configure-vsts-source.component.ts

```diff
@@ -140,7 +140,7 @@ export class ConfigureVstsSourceComponent implements OnDestroy {
           this._vstsRepositories.map(repo => {
             return {
               displayLabel: `${repo.project.name} (${repo.project.id})`,
-              value: repo.project.id,
+              value: repo.project.name,
```

Comment:
Please check if we surface this value on summary page or not. If yes, then verify once if there is any effect of this over the value we are surfacing on Summary page.

Reply:
We surface project name in the repository url and I have checked that it works fine.